ras/smtstate.py

```diff
@@ -27,7 +27,7 @@ class smtstate_tool(Test):
     def setUp(self):
 
         sm = SoftwareManager()
-        self.time_in_minutes = self.params.get('time_in_minutes', default=5)
+        self.time_in_minutes = self.params.get('time_in_seconds', default=300)
```

Comment:
can you please make this variable scope is local

Reply:
sure